wgpu/gui/auto.py

```diff
@@ -27,9 +28,35 @@ def is_jupyter():
 else:
     try:
         from .glfw import WgpuCanvas, run, call_later  # noqa
-    except ImportError as err:
-        msg = str(err)
-        msg += "\n\n  Install glfw using e.g. ``pip install -U glfw``."
-        if sys.platform.startswith("linux"):
-            msg += "\n  You may also need to run the equivalent of ``apt install libglfw3``."
-        raise ImportError(msg) from None
+    except ImportError as glfw_err:
+        try:
+            for libname in ("PySide6", "PyQt6", "PySide2", "PyQt5", "PySide", "PyQt4"):
+                try:
+                    importlib.import_module(libname)
+                    break
+                except ImportError:
+                    pass
+
+            from .qt import WgpuCanvas, QtWidgets, QtCore
+
+            # When using Qt, there needs to be an
+            # application before any widget is created
+            app = QtWidgets.QApplication([])
+
+            def run():
+                app.exec() if hasattr(app, "exec") else app.exec_()
+
+            def call_later(delay, callback, *args):
+                QtCore.QTimer.singleShot(delay * 1000, lambda: callback(*args))
+
+        except ImportError as qt_err:
```

Comment:
Does this catch exceptions raised from `from .qt import ...`? I think we can move this part to the else clause of the try-except that uses `importlib.import_module(libname)`.

Reply:
I didn't really get what you were after with this comment, but I think I managed to address your concern. I moved it to the else clause of the for loop instead though.